<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>800</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Data Recorder</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_0">
            <item>
                <widget class="QGroupBox" name="grpBoxDataRecorder">
                    <property name="title">
                        <string>Data Recoder</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="0" column="3">
                            <layout class="QVBoxLayout" name="verticalLayout_2">
                                <item alignment="Qt::AlignRight|Qt::AlignCenter">
                                    <widget class="QLabel" name="lblMessageRecorder">
                                        <property name="text">
                                            <string>&lt;Message&gt;</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="0" column="1">
                            <layout class="QHBoxLayout" name="horizontalLayout_3">
                                <item>
                                    <widget class="QLabel" name="lblCurrentDataFile">
                                        <property name="text">
                                            <string>Current data file: </string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="lblDataFilename">
                                        <property name="text">
                                            <string>&lt; none &gt;</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="0" column="0">
                            <widget class="QPushButton" name="btnInitialize">
                                <property name="minimumSize">
                                    <size>
                                        <width>100</width>
                                        <height>50</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>Initialize</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="grpBoxDataRecorderSettings">
                    <property name="title">
                        <string>Data Recoder Settings</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_items"></layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
